tools/ssh-bastion-ec2/ssh-bastion-ec2

```diff
@@ -117,12 +119,12 @@ def ssh_ec2(remote_host, bastion_ip, key):
   retVal = 1
   try:
     tmp_key_path, ssh_cmd = _get_ssh_command(remote_host, bastion_ip, key)
-    # print("SSH_COMMAND: ", ssh_cmd)
-    retVal = os.system(ssh_cmd)
+    logging.info("SSH_COMMAND: %s", ' '.join(ssh_cmd))
+    retVal = subprocess.call(ssh_cmd)
   except Exception as e:
     # It is bad practice to except on any exception but we want to remove
     # the tmp ssh key regardless of what might happen.
-    print("ssh failed with exception: {}".format(e))
+    logging.exception("SSH failed with exception: ")
```

Comment:
It says "with exception:" but `e` is not being stringified.

Reply:
Why testing, the stringified execption was bring printed to the console without adding it explicitly to the exception.